crates/bin/pcli/src/command/tx/liquidity_position.rs

```diff
@@ -104,6 +104,9 @@ pub enum OrderCmd {
         /// The selected fee tier to multiply the fee amount by.
         #[clap(short, long, default_value_t)]
         fee_tier: FeeTier,
+        /// Duplicate the order for the given number of times.
+        #[clap(short, long, default_value = "1")]
+        duplicate: u32,
```

Comment:
IMO we should change this to something starting with `n` to match the dutch auctions